### DISCUSSION TITLE
Meeting title color contrast

### DISCUSSION THREAD

**Discussion by D43-web** [code4recovery/12-step-meeting-list#1458] [Q&A] [answered]
I want to be sure our the Meeting page on our district website is clear and there is enough contrast between the text and the background so that it is accessible to all (ie. colorblind, low vision). I am concerned the Meeting names, which are a blue link, are not dark enough.
url: https://aa-nia-dist43.org/meetings/
Divi theme 
wordpress version 6.5.5
12 step meeting list version 3.14.35
I tried applying #tsml { color: black; } in 'Customizing' > Additional CSS, but it did not appear to change anything. This was from a search for a similar discussion. The answer that worked in that discussion did not work for me.
I appreciate any help you can offer. I am new to web admin and I don't code, so simple explanations are appreciated. Thank you, we really love your product. It's brilliant!

**Comment by joshreisner** [comment, marked as answer]
yah, where you set the black color also `background: white;` - this should really bring out the contrast. or you could try the TSML UI appearance, that might also fix it, but i'm not 100% sure

**Reply by gkovats** [reply]
Can also try this CSS, changes even rows to a light gray
```
#tsml .table-striped>tbody>tr:nth-of-type(even) { background-color: #eee; }
```

**Reply by D43-web** [reply]
I changed to the TSML UI, and I really like it. The contrast is much better.
Thank you for all the helpful suggestions.

**Comment by SIA-WEBSUP** [comment]
You might also consider changing the  anchor text color to a darker blue, or use a bold font, or some combination of all these attributes.

`#tsmal a { color: #1E3A8A; font-weight:600 }`
